tests/address.test.js

```diff
@@ -444,6 +440,87 @@ describe('Address', () => {
     })
   })
 
+  describe('addOrReplaceDeliveryAddressOnList', () => {
+    it('should be empty if empty params are passed', () => {
+      const addresses1 = addOrReplaceDeliveryAddressOnList()
+      const addresses2 = addOrReplaceDeliveryAddressOnList([], null)
+
+      expect(addresses1).toBeUndefined()
+      expect(addresses2).toEqual([])
+    })
+
+    it('should add address on empty address list', () => {
+      const residentialAddress1 = addresses.residentialAddress
+      const addresses1 = []
+      const expectedAddresses1 = [...addresses1, residentialAddress1]
+
+      const resultAddresses1 = addOrReplaceDeliveryAddressOnList(
+        addresses1,
+        residentialAddress1
+      )
+
+      expect(resultAddresses1).toEqual(expectedAddresses1)
```

Comment:
Why do you use `samename1` when you do not have a `samename2`?

```suggestion
      const { residentialAddress } = addresses
      const addresses = []
      const expectedAddresses = [...addresses, residentialAddress]

      const resultAddresses = addOrReplaceDeliveryAddressOnList(
        addresses,
        residentialAddress
      )

      expect(resultAddresses).toEqual(expectedAddresses)
```

Reply:
Not sure, but the Pattern on this file was that one